rpm/podman.spec

```diff
@@ -35,7 +35,7 @@
 %global git0 %{container_base_url}/%{name}
 
 # podman-machine subpackage will be present only on these architectures
-%global machine_arches x86_64 aarch64
+%global machine_arches x86_64 aarch64 riscv64
```

Comment:
This seems wrong, podman machine does not support riscv64

Reply:
You are right. I made the change mechanically, without testing machines. Force pushed without it